### ISSUE TITLE
Package rule false positive when there is no package

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#809]
rules 2.8.2 tested in Sonarlint

project sources are in src, custom.cls is directly in src, no subdirectory = no package

This class shows 2 false positives

* Package name  doesn't match directory name
* #591 or similar,  ribbonTabStart  is flagged as unused

```
using Infragistics.Win.UltraWinToolbars.UltraToolbarsManager from assembly.

block-level on error undo, throw.

class custom:

        define variable ribbonTabStart as Infragistics.Win.UltraWinToolbars.RibbonTab no-undo.
        define variable ultraToolbarsManager1 as UltraToolbarsManager no-undo.

    method public void Test():

        ribbonTabStart = ultraToolbarsManager1:Ribbon:Tabs:Item['Start'].
        this-object:ribbonTabStart:Groups:Item['Volgorde']:Visible = false .

    end method.

end class.
```

**Comment by gquerret** [comment]
Implementation was changed in 2.8, there was clearly a missing test case ! Will be fixed next week.

**Comment by gquerret** [comment]
Fxied
